sdcm/utils/data_validator.py

```diff
@@ -251,7 +251,7 @@ def get_view_name_from_profile(self, name_substr, cs_profile=None, all_entries=F
         mv_names = []
 
         if not cs_profile:
-            stress_cmd = self.longevity_self_object.params.get(self.stress_cmds_part, default=[])
+            stress_cmd = self.longevity_self_object.params.get(self.stress_cmds_part) or []
```

Comment:
why do you keep default here?

Reply:
Expected list there. If there is no stress commands in this part, empty list will be at least